### PR TITLE
Add Truffleruby head to CI

### PR DESCRIPTION
**Pull request by gogainda**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,19 +19,14 @@ jobs:
         ruby:
           - 3.1.2
           - 3.0.4
-          - 2.7.6
-          - 2.6.10
-          - 2.5.8
```
Comment (santib) [inline .github/workflows/build.yml:24]: I don't think this PR should remove other ruby/rails versions from the CI

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -19,19 +19,14 @@ jobs:
...
+          - truffleruby-head
```
Comment (santib) [inline .github/workflows/build.yml:22]: I don't think we want to test against head, we don't even do that with MRI. Maybe we could add some stable versions instead

Comment (grzuy) [reply .github/workflows/build.yml:22]: Agree.

**Comment by grzuy** [comment]
Sorry this became a bit stale because of my delay.

Feel free to re-open with a stable version of truffleruby.

Thanks @gogainda.

**Pull request closed without merging**